### ISSUE TITLE
[Critical] Shared M4A audio clips do not produce sound when played by the sender or the recipient(s)

### ISSUE THREAD

**Issue by levicki** [TelegramMessenger/Telegram-iOS#1045]
### Checklist
- [x] I am reporting an issue in existing functionality that does not work as intended
- [x] I've searched for existing [GitHub issues](https://github.com/telegrammessenger/Telegram-iOS/issues)

### Description
Audio clips in `.m4a` format shared to Telegram groups or individual chats do not produce any sound when played by the sender or the recipient(s).

When the clip is shared directly from Voice Memos, the waveform is shown in Telegram but there is no sound when you try to play it.

When the clip is shared by sending the audio file to the Windows PC using other means, and the resulting .m4a file is then shared as an attachment using Telegram for Windows (drag and drop on chat or group), the resulting file still does not play any sound in Telegram for iOS.

All files shared by any means mentioned play in Telegram Desktop for Windows version just fine.

If you select the shared audio clip in Telegram for iOS and then select Save to Files... from the share sheet, the resulting file plays just fine from Files app on the phone.

TL;DR &mdash; M4A file playback is broken in Telegram for iOS.

Please fix.

### Expected Behavior
Sound should be playing.

### Actual Behavior
Sound isn't playing.

### Steps to Reproduce
1. Record an audio clip on an iPhone using Voice Memos
2. Share it to a Telegram chat
3. Try playing it

### Screenshots and Videos 
N/A

### Environment
**Device:** `iPhone 13 Pro Max`

**iOS version**: `16.3.1`

**App version:** `9.4.3`

**Comment by levicki** [comment]
I am getting reports that the same prioblem exists on Android so it might be some upstream issue.

**Comment by levicki** [comment]
@laktyushin Could someone please investigate this? If you need additional information please let me know. I can also provide audio file if necessary.

**Comment by ali-fareed** [comment]
Could you please attach a sample file?

**Comment by laktyushin** [comment]
@levicki It plays for me without any issues on 14, 13 and 11 Pro Maxes with iOS 16.3.1 and Telegram 9.5.
Any ideas what is different in your case? Are you sure the volume is not turned off? Are headphones being used?

**Comment by levicki** [comment]
@laktyushin I wrote this bug report for the application version **9.4.3** which for me didn't play the file when shared to Telegram chat or group. Volume is not turned off because I can record audio in Telegram and share it and that plays just fine. Headphones were not used by any of the participants.

I just updated to 9.5.1 and now it plays the file. I assume it didn't magically fix itself?

**Comment by laktyushin** [comment]
You're right, it was an old issue and 9.4.3 didn't have the fix yet.

I haven't seen #1040 yet, thanks for pointing it out.

**Comment by levicki** [comment]
@laktyushin In 9.5 on Android sound doesn't play.

**Comment by laktyushin** [comment]
@levicki Yes, we're only responsible for the iOS app here. For Android you can create a bug report on the platform – bugs.telegram.org 
I don't see any open cards about M4A there at the moment.
